inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx

```diff
@@ -159,27 +246,133 @@ export class Timeline extends React.Component<
     this.setState({ activeKeyframe: null });
   }
 
+  private active: string = '';
+
   scrollDragStart(e: React.TouchEvent<HTMLDivElement>): void;
   scrollDragStart(e: React.MouseEvent<HTMLDivElement, MouseEvent>): void;
   scrollDragStart(e: any) {
     e.preventDefault();
-    if ((e.target.class = 'scrollbar') && this._scrollbarHandle.current) {
-      this._scrolling = true;
+    if (e.target.className === 'scrollbar') {

```

Comment:
The class check looks redundant

Reply:
let me check again. But I remember adding this because I'm using a container to set dragging events but I'm actually moving a child...

Reply:
yes, that was redundant 👍 , I removed it , thanks for the review.